spec/features/user_can_reply_to_a_twit_spec.rb

```diff
@@ -0,0 +1,71 @@
+require "rails_helper"
+
+RSpec.feature "Reply to a Twit", type: :feature do
+  before do
+    # Short circuit OmniAuth requests to use a mock authentication hash
+    OmniAuth.config.test_mode = true
+
+    OmniAuth.config.mock_auth[:twitter] = OmniAuth::AuthHash.new({
+      :provider => "twitter",
+      :uid => "123456",
+      :info => {
+        :nickname => "imwithsam",
+        :name => "Samson Brock",
+        :location => "Denver, CO",
+        :image => "https://pbs.twimg.com/profile_images/606479431251550208/uYn3hNom.jpg",
+        :description => "Turing School of Software & Design Student.",
+        :urls => {
+          :Website => "http://badmotivator.io/",
+          :Twitter => "https://twitter.com/imwithsam"
+        }
+      },
+      :credentials => {
+        :token => ENV["twitter_token"],
+        :secret => ENV["twitter_token_secret"]
+      }
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.